junit-platform-console/junit-platform-console.gradle

```diff
@@ -33,18 +35,26 @@ idea {
 	}
 }
 
-mainClassName = "org.junit.platform.console.ConsoleLauncher"
+jar {
+	manifest {
+		attributes(
+			'Main-Class': 'org.junit.platform.console.ConsoleLauncher',
+			// generate test engine version information in single shared manifest file
+			// pattern of key and value: `'Engine-Version-{YourTestEngine#getId()}': '47.11'`
+			'Engine-Version-junit-jupiter': "$rootProject.version",
+			'Engine-Version-junit-vintage': "$vintageVersion"
+		)
+	}
+}
 
 javadoc {
 	classpath = project.sourceSets.main.compileClasspath + configurations.shadowed
 }
 
-jar.enabled = false
-
 shadowJar {
-	classifier = null
+	classifier = 'standalone'
```

Comment:
Rephrasing your requested change in my words: "create two shadowed jars"

- _normal_ `junit-platform-console-1.0.0-SNAPSHOT.jar`
containing "junit-platform-console" classes and " jopt-simple"

- _standalone_ `junit-platform-console-1.0.0-SNAPSHOT-standalone.jar`
containing "junit-platform-console" classes", " jopt-simple", Jupiter and Vintage engine and **all** other dependencies like "platform-commons", "platform-engine"...